enhancements/two-nodes-openshift/2no.md

```diff
@@ -300,26 +369,29 @@ This proposal is an alternative architecture to Single-node and MicroShift, so i
 
 1. Risk: If etcd were to be made active on both peers during a network split, divergent datasets would be created
    1. Mitigation: RHEL-HA requires fencing of a presumed dead peer before restarting etcd as a cluster-of-one
-   1. Mitigation: Peers remain inert (unable to fence peers, or start cri-o, kubelet, or etcd) after rebooting until they can contact their peer
+   2. Mitigation: Peers remain inert (unable to fence peers, or start cri-o, kubelet, or etcd) after rebooting until they can contact their peer
 
-1. Risk: Multiple entities (RHEL-HA, CEO) attempting to manage etcd membership would cause an internal split-brain
+2. Risk: Multiple entities (RHEL-HA, CEO) attempting to manage etcd membership would cause an internal split-brain
    1. Mitigation: The CEO will run in a mode that does manage not etcd membership
 
-1. Risk: Rebooting the surviving peer would require human intervention before the cluster starts, increasing downtime and creating an admin burden at remote sites
+3. Risk: Other operators that perform power-management functions could conflict with pacemaker.
+   1. Mitigation: Update the Baremetal and Node Health Check operators to ensure control plane nodes can not perform power operations in the 2-node topology.
+
+4. Risk: Rebooting the surviving peer would require human intervention before the cluster starts, increasing downtime and creating an admin burden at remote sites
    1. Mitigation: Lifecycle events, such as upgrades and applying new `MachineConfig`s, are not permitted in a single-node degraded state
-   1. Mitigation: Usage of the MCO Admin Defined Node Disruption [feature](https://github.com/openshift/enhancements/pull/1525) will further reduce the need for reboots.
-   1. Mitigation: The node will be reachable via SSH and the confirmation can be scripted
-   1. Mitigation: It may be possible to identify scenarios where, for a known hardware topology, it is safe to allow the node to proceed automatically.
+   2. Mitigation: Usage of the MCO Admin Defined Node Disruption [feature](https://github.com/openshift/enhancements/pull/1525) will further reduce the need for reboots.
+   3. Mitigation: The node will be reachable via SSH and the confirmation can be scripted
+   4. Mitigation: It may be possible to identify scenarios where, for a known hardware topology, it is safe to allow the node to proceed automatically.
 
-1. Risk: “Something changed, let's reboot” is somewhat baked into OCP’s DNA and has the potential to be problematic when nodes are actively watching for their peer to disappear, and have an obligation to promptly act on that disappearance by power cycling them.
+5. Risk: “Something changed, let's reboot” is somewhat baked into OCP’s DNA and has the potential to be problematic when nodes are actively watching for their peer to disappear, and have an obligation to promptly act on that disappearance by power cycling them.
```

Comment:
Might worth mentioning that most of those events (maybe even all of them :thinking: ?) falls into the [first bucket](https://github.com/mshitrit/enhancements/pull/3/files#diff-f4f2dd40af3dfd355440cc5094e2c4d21877db4e5cd0d1c6830ad7b9eb84186cR275) mentioned, which means it's already handled.